test/WebJobs.Script.Tests/ScriptHostTests.cs

```diff
@@ -62,7 +62,7 @@ public void ReadFunctionMetadata_Succeeds()
             var traceWriter = new TestTraceWriter(TraceLevel.Verbose);
             var functionErrors = new Dictionary<string, Collection<string>>();
             var metadata = ScriptHost.ReadFunctionMetadata(config, traceWriter, null, functionErrors);
-            Assert.Equal(40, metadata.Count);
+            Assert.Equal(52, metadata.Count);
```

Comment:
This is an unfortunate side effect of removing the excluded flag. A lot of the functions that we didn't run while debugging before will be executing now.

We should be able to maintain the same behavior by adding the appropriate disabled flags to `appsettings.json`

Reply:
Would still like to make sure we're addressing this. Some suggestions in the comment about the exclude flag above.